### ISSUE TITLE
Allow running vpl_evaluate.sh without chroot

### ISSUE THREAD

**Issue by aivarannamaa** [jcrodriguez-dis/vpl-xmlrpc-jail#9]
At the moment vpl_evaluate.sh seems to be run with the same restricted environment than vpl_execution. 

Is it really necessary? If vpl_evaluete.sh could be run under a normal non-root user, one could implement eg. evaluation logging or some other tricks that are not possible under chroot.

What do you think about adding a configuration option to indicate username and password for running vpl_evaluate.sh ?

**Comment by jcrodriguez-dis** [comment]
Yes, vpl_evaluate.sh, vpl_run.sh and vpl_debug.sh run with the same restricted environment than vpl_execution.

I think that security must be the main feature. I dont know the full effects of using a common user outside the chroot to compile the code.

At a first glance I think that it may be a bad solution to your needs of persistent log of evaluation. For example: this solutions is in confict with the use of multiple jail servers. The logs are locals to each server.

Nevertheless, I need to think about your need, to find a solution simple, clear, secure and of general purpose.
Best regards.

**Comment by aivarannamaa** [comment]
I proposed a less general (and much less complex) solution: https://github.com/jcrodriguez-dis/vpl-xmlrpc-jail/issues/11
